apps/server/src/core/error/filter/global-error.filter.ts

```diff
@@ -1,136 +1,31 @@
-import { ArgumentsHost, Catch, ExceptionFilter, HttpException, InternalServerErrorException } from '@nestjs/common';
```

Comment:
Fehlen hier noch index files? Oder fügen wir die erst hinzu wenn es mehrere Dateien sind?

Reply:
Weiß ich gar nicht, wie das sein soll. Ich hatte auch überlegt, den Filter eine Ebene höher direkt in den error-Ordner zu legen, weil er ja der Kern des Moduls ist....